future-apps/payroll/contracts/Payroll.sol

```diff
@@ -737,18 +737,13 @@ contract Payroll is EtherTokenConstant, IForwarder, IsContract, AragonApp {
     function _removeEmployeeIfTerminatedAndPaidOut(uint256 _employeeId) internal {
         Employee storage employee = employees[_employeeId];
 
-        if (employee.endDate > getTimestamp64()) {
-            return;
-        }
-        if (_getCurrentCappedOwedSalary(_employeeId) > 0) {
-            return;
+        if (
+            employee.lastPayroll == employee.endDate &&
```

Comment:
Note that this change requires 3eab68a, to ensure that `lastPayroll` is set to `endDate` when drawing out all funds from a terminated employee.

Reply:
I see and I like the idea